### PR TITLE
Update Classic Notifications for Activity page changes

### LINKED ISSUE #1318: Classic Notifications broken by the new Activities update
Scope of brokenness is unknown, but we've had two people in chat asking about it.

tagging #1312 for metabug purposes

### PR DESCRIPTION
**Pull request by hobinjk**
Fixes #1318.

Main changes are:
- `part_*` classes becoming `activity-notification__activity_*`
- `stage` becomes `activity-notification` (kinda)

### CODE REVIEW

File: Extensions/old_notifications.js [hunk 1]
```diff
@@ -64,15 +64,18 @@ XKit.extensions.old_notifications = new Object({
 
 	render_notification: function(obj) {
 
-		if ($(obj).find(".ui_note").length <= 0) { return ""; }
+		obj = $(obj);
```
Comment (nightpool) [inline Extensions/old_notifications.js]: convention is to use a prefixed $ to indicate a jQuery variable.

File: Extensions/old_notifications.js [hunk 2]
```diff
@@ -121,23 +124,23 @@ XKit.extensions.old_notifications = new Object({
...
-		var is_like = $(obj).hasClass("is_like");
-		var is_follow = $(obj).hasClass("is_follower");
-		var is_mention = $(obj).hasClass("is_user_mention");
+		var is_friend = obj.hasClass("is_friend");
+		var is_reblog = obj.hasClass("is_reblog");
+		var is_reply = obj.hasClass("is_reply");
+		var is_answer = obj.hasClass("is_answer");
+		var is_like = obj.hasClass("is_like");
+		var is_follow = obj.hasClass("is_follower");
+		var is_mention = obj.hasClass("is_user_mention");
```
Comment (nightpool) [inline Extensions/old_notifications.js]: this changed to `user_mention`.

Comment (nightpool) [inline Extensions/old_notifications.js]: There's also a new type, `note_mention`, for when someone @-mentions you in a *reply* to a post. I would suggest including that in the same selector.

File: Extensions/old_notifications.js [hunk 3]
```diff
@@ -185,6 +188,11 @@ XKit.extensions.old_notifications = new Object({
 		var m_inner_html = "";
 		var no_preview_frame = false;
 
+		if (obj.hasClass('fanmail')) {
```
Comment (nightpool) [inline Extensions/old_notifications.js]: holy shit

File: Extensions/old_notifications.js (hunk 2 above)
Comment (hobinjk) [inline Extensions/old_notifications.js]: Thanks for the info! Updated accordingly

File: Extensions/old_notifications.js (hunk 1 above)
Comment (hobinjk) [inline Extensions/old_notifications.js]: Changed, I'd like to take this time to thank eslint for it's `no-undef` rule

**Comment by nightpool** [comment]
I don't know if this is a regression or not, but ask notifications are broken:

image

image

**Comment by nightpool** [comment]
I'm going to merge and file a followup bug, since it seems pretty minor.

**Pull request merged**